### DISCUSSION TITLE
`Something went wrong` visiting deleted account

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#65601] [Mobile] [answered]
GitHub Mobile Android beta

1. Report an account for abuse - https://github.com/Tarrexino
2. Get reply from support hinting that they deleted the account
3. Click the link in the email from the original submission about https://github.com/Tarrexino

![Screenshot_20230831-120248.png](https://github.com/community/community/assets/2119212/56521523-f969-4a3d-8259-f5439733daf6)

Expected results: if an account doesn't exist (because it's deleted/doesn't exist today), yield an error saying there's no account by that name.

**Comment by alcere** [comment, marked as answer]
Hi @jsoref, thank you very much for reporting this issue! I will create an internal issue to follow up on this and change the behaviour!

**Reply by bautistaramir0714** [reply]
My account is deleted I'm trying to figure out this issue thank you..

**Reply by jsoref** [reply]
This isn't the right place for you.

Generally accounts are deleted because either:
* someone controlling the account deletes them
* someone controlling the account performed actions in violation of the terms of service

In both cases, you can talk to GitHub support, but you'll mostly want to make sure your account is used appropriately.
